doc/developer-guide/developer-guide.rst

```diff
@@ -25,7 +25,7 @@ Architecture
 
 The classes in module are listed below.
 
-.. uml:: ../uml/m2_class.uml
+.. mermaid:: ../uml/m2_class.uml
```

Comment:
The mermaid web recommends either .mermaid or .mmd - can you also rename the file to m2_class.mmd and change the include here?

Reply:
Rename to `.mmd` in the commit: [32d73c4](https://github.com/lsst-ts/ts_m2/pull/61/commits/32d73c4e95ccdcb3cea89970dbd559d0811bcdf5). Thanks!